index.bs

```diff
@@ -22,11 +22,9 @@ Markup Shorthands: css no
 Logo: https://webmachinelearning.github.io/webmachinelearning-logo.png
 Deadline: 2023-10-01
 Status Text: <p>
-  Further implementation experience and user feedback is being gathered for the
-  {{MLCommandEncoder}} interface that proposes to enable more efficient WebGPU
-  integration. A proposal to
-  <a href="https://github.com/webmachinelearning/webnn/pull/322">simplify
-  MLContext creation</a> is being discussed. This document is maintained and
+  Since the <a href="https://www.w3.org/TR/2023/CR-webnn-20230330/">initial Candidate Recommendation Snapshot</a> the Working Group has gathered further <a href="https://webmachinelearning.github.io/webnn-status/">implementation experience</a> and added new operations and data types needed for well-known <a href="https://github.com/webmachinelearning/webnn/issues/375">transformers to support generative AI use cases</a>. In addition, informed by this implementation experience, the group removed <code>MLCommandEncoder</code></a>, support for synchronous execution, and higher-level operations that can be expressed in terms of lower-level primitives in a performant manner. The group has also updated the specification to use modern authoring conventions to improve interoperability and precision of normative definitions and is developing a new feature, a <a href="https://github.com/webmachinelearning/webnn/issues/482">backend-agnostic storage type</a>, to improve performance and interoperability between the WebNN, WebGPU APIs and purpose-built hardware for ML.
```

Comment:
Thinking some more about the backend-agnostic storage new feature - I think we should indicate whether we expect to republish a CR Snapshot if/when that feature lands (vs keeping it for a potential post-Recommendation iteration); I'm happy to propose some suggested language if you confirm the former is our plan.

Reply:
I believe we don’t want to rush to post-CR stages, especially considering more implementers just joined the WG. I’d expect us to do another CRS when this new feature has matured adequately.

Contributions welcome.